.goreleaser.yml

```diff
@@ -0,0 +1,52 @@
+project_name: driftctl
+dist: bin
+release:
+    github:
+        owner: cloudskiff
+        name: driftctl
+    draft: false
+    prerelease: auto
+before:
+    hooks:
+        - go mod tidy
+        - go generate ./...
+signs:
+    - signature: "${artifact}.gpg"
+      artifacts: checksum
+      args: [ "--batch", "-u", "{{ .Env.SIGNINGKEY }}", "--output", "${signature}", "--detach-sign", "${artifact}" ]
```

Comment:
We expect the Key ID or email instead `{{ .Env.SIGNINGKEY }}`. This env contains the whole private key that should be decoded from base64 (see my comment on the circleci config file)

We should have this here `args: [ "--batch", "-u", "security@cloudskiff.com", "--output", "${signature}", "--detach-sign", "${artifact}" ]`

Reply:
copy/paste mistake, it was supposed to be the fingerprint